ammo-fe/cypress/integration/TopBar.cy.ts

```diff
@@ -0,0 +1,65 @@
+import WS from '../../src/network/WS';
+import { bulletMock } from '../../src/shared/mocks/Bullet.mock';
+
+const socket = WS.getSocket('http://localhost:3001');
+
+describe('Top bar', () => {
+    before(() => {
+        cy.visit('/');
+    });
+
+    it('Recorder button is clickable as we are connected', () => {
+        cy.get('[data-cy=recorder-button]').should('be.enabled');
+    });
+
+    it('Recorder button first image must be pause', () => {
+        cy.get('[data-cy=recording-button-pause]').should('not.be.visible');
+    });
```

Comment:
I'd add a test to verify that the recording works in the default state (before even clicking on the button)